index.ts

```diff
@@ -44,14 +44,14 @@ export class DataLoaderInterceptor implements NestInterceptor {
     const ctx: any = graphqlExecutionContext.getContext();
 
     if (ctx[NEST_LOADER_CONTEXT_KEY] === undefined) {
-      ctx[NEST_LOADER_CONTEXT_KEY] = (type: string): NestDataLoader<any, any> => {
```

Comment:
Can the return type of this method remain specified?

Reply:
Yes! I put back the return type here. Thank you for the comment!